cleverhans/attacks_tf.py

```diff
@@ -57,6 +63,38 @@ def fgsm(x, predictions, eps, clip_min=None, clip_max=None):
     return adv_x
 
 
+def fgsm_iterative(x, y, model, eps, eps_iter, n_iter,
```

Comment:
maybe "basic_iterative" instead to match Alex's naming? The "f" in "fgsm" doesn't apply because it meant "fast" compared to iterative methods